core/src/main/scala/org/apache/spark/h2o/converters/H2ORDDLike.scala

```diff
@@ -23,7 +23,8 @@ import water.fvec.{Frame, FrameUtils}
 /**
  * Contains functions that are shared between all H2ORDD types (i.e., Scala, Java)
  */
-private[converters] trait H2ORDDLike[T <: Frame] {
```

Comment:
yup, I'm good with restriction like that, but still don't see why moving `H2OContext` somewhere. What are your intentions @vpatryshev with this please ?